src/WooCommerce.php

```diff
@@ -9,6 +9,7 @@ class WooCommerce {
 
   const FIELD_MARKETING_FOCUS = '_' . Plugin::PREFIX . '_marketing_focus';
   const FIELD_PRICE_COMPARISON_FOCUS = '_' . Plugin::PREFIX . '_price_comparison_focus';
+  const FIELD_PRICE_COMPARISON_LOWER_ONLY = '_' . Plugin::PREFIX . '_price_comparison_lower_only';
```

Comment:
The field name is not really self-descriptive.  Maybe the "comparison" part can be skipped?

```suggestion
  const FIELD_REPRICING_LOWER_ONLY = '_' . Plugin::PREFIX . '_repricing_lower_only';
```